docs/further.md

```diff
@@ -180,38 +177,34 @@ You can check whether your cluster has this enabled with:
+
+If your cluster does not support automated requeing, you can ask the plugin to requeu preemted jobs with the `--slurm-requeue` flag: 
 
 ```console
 snakemake --slurm-requeue ...
 ```
 
-This might be the default on your cluster, already.
-TODO: What exactly might be the default on a cluster?
-TODO: Also, what exactly does the `--slurm-requeue` mode do? I assume it makes snakemake handle the requeueing instead of slurm doing this internally?
-
+This flag effectively does not consider failed SLURM jobs or preserves job IDs and priorities or allows job priority to be accumulated while pending.
 
-#### MPI-specific resources
 
-TODO: From what I can see, snakemake-executor-plugin-slurm does not do anything special to mpi jobs. So it might be sufficient to point to the main snakemake MPI support docs at https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html and maybe extend those (or clean them up). The only Slurm-specific info is probably to use `mpi="srun"` instead of `mpi="mpiexec"`, right? But maybe I am also overlooking any special things happening in the software package stack of the plugin (or the jobstep)?
+#### MPI-specific Resources
 
 Snakemake's SLURM executor plugin supports the execution of MPI ([Message Passing Interface](https://en.wikipedia.org/wiki/Message_Passing_Interface)) jobs.
-Per default, jobs can only run on a single cluster node (or machine) and parallelization is thus limited by the maximum number of cores that is available on any machine in the cluster.
-MPI jobs enable parallel computations spanning across multiple nodes, thus potentially parallelizing to more cores than any machine in you cluster can offer.
+
+Per default, jobs can only run on a single cluster node (or machine) and parallelization is thus limited by the maximum number of cores that is available on any machine in the cluster. MPI jobs enable parallel computations spanning across multiple nodes, thus potentially parallelizing to more cores than any machine in you cluster can offer. [See the main documentation section of Snakemake, too.](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#mpi-support)
 
 | Snakemake     | Description                                 |
 |---------------|---------------------------------------------|
 | `mpi`         | MPI launcher command, for example `srun`    |
-| `tasks`       | the walltime per job in minutes             |
+| `tasks`       | The number of SLURM tasks - equivalent to MPI-ranks |
 
 
-To effectively utilize MPI within a Snakemake workflow, it's recommended to use `srun` as the MPI launcher when operating in a SLURM environment.
+To effectively utilize MPI within a Snakemake workflow, it's recommended to use `srun` as the MPI launcher when operating in a SLURM environment. However, some programs do not work well with this MPI launcer or require a detailed topology layout - this can be added to the `srun` statement, if required.
```

Comment:
Again, one line per sentence for better diffs:
```suggestion
To effectively utilize MPI within a Snakemake workflow, it's recommended to use `srun` as the MPI launcher when operating in a SLURM environment.
However, some programs do not work well with this MPI launcer or require a detailed topology layout - this can be added to the `srun` statement, if required.
```